pxtblocks/blocklyloader.ts

```diff
@@ -390,9 +390,12 @@ namespace pxt.blocks {
         }
 
         let hash = JSON.stringify(fn);
+        /* tslint:disable:possible-timing-attack */
+        // TODO(tslint): look into this possible timing attack
```

Comment:
this is not a cryptographic related codepath, there is no point on doing an attack here.

Reply:
Okay, will remove TODO.